app/src/pages/inside/stepPage/modals/editDefectModals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -67,7 +67,9 @@ const MakeDecision = ({ data }) => {
   const [issueAction, setIssueAction] = useState({});
 
   useEffect(() => {
-    setModalHasChanges(!isEqual(itemData.issue, state.issue) || !isEmptyObject(issueAction));
+    setModalHasChanges(
+      state.issue ? !isEqual(itemData.issue, state.issue) || !isEmptyObject(issueAction) : false,
```

Comment:
The user should have possibility to interact with issue action even when there is no issue in the state.
Also notice how the item data gets to the next action.